src/main/java/lotto/domain/Ranking.java

```diff
@@ -0,0 +1,62 @@
+package lotto.domain;
+
+import java.util.Arrays;
+
+public enum Ranking {
+    FIRST(6, 0, 2000000000),
+    SECOND(5, 1, 30000000),
+    THIRD(5, 0, 1500000),
+    FOURTH(4, 0, 50000),
+    FIFTH(3, 0, 5000),
+    OTHERS(0, 0, 0);
+
+    private final int numberOfMatches;
+    private final int prizeMoney;
+    private final int bonus;
+    private int count = 0;
+
+    Ranking(final int numberOfMatches, int bonus, final int prizeMoney) {
+        this.numberOfMatches = numberOfMatches;
+        if (numberOfMatches == 5) {
+            this.bonus = bonus;
+        } else {
+            this.bonus = 0;
+        }
+        this.prizeMoney = prizeMoney;
+    }
+
+    public static Ranking of(final int numberOfMatches, final int bonus) {
+        return Arrays.stream(values())
+            .filter(
+                Ranking -> (Ranking.numberOfMatches == numberOfMatches) &&
+                    (Ranking.bonus == bonus))
```

Comment:
filter 내부의 람다식 변수도 지역 변수처럼 변수명을 `lowerCamelCase`로 작성해주세요! 🙏
```suggestion
                ranking -> (ranking.numberOfMatches == numberOfMatches) &&
                    (ranking.bonus == bonus))
```